<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera</class>
 <widget class="QMainWindow" name="Camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>298</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/resources/images/title.jpg</normaloff>:/new/prefix1/resources/images/title.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <widget class="QWidget" name="pagePrev">
     <widget class="QVideoWidget" name="videoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageLast">
     <widget class="QLabel" name="labelImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>391</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Recorde Time(s):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recorde_time">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="CameraPage">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Cam">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="btn_camera_start">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_capture">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Capture</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/new/prefix1/resources/images/icon.ico</normaloff>:/new/prefix1/resources/images/icon.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_camera_stop">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="lab_resl">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>601 x 298</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_camera_setting">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Video">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="btn_record_stop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record_pause">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record_muted">
      <property name="geometry">
       <rect>
        <x>99</x>
        <y>70</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Muted</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_record_metaData">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Set MetaData</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_video_setting">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_path">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
